vendor/vendor.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Don't commit the `vendor.json` file into the repo. Sadly in Go there is no clean way to vendor dependencies for libraries as it messes up with the build.